### PR TITLE
encyrpt PII using lit protocol

### PR DESCRIPTION
**Pull request by gravityblast**
* implemented PII encryption using lit protocol
* set the `encrypted` field to `true` to questions with words `email` or `cool`. this is temporarily until we have a round with the final metadata
* temporarily commented out the application builder tests until we can fix an error caused by the lit js sdk

### CODE REVIEW

File: client/src/actions/roundApplication.ts [hunk 1]
```diff
@@ -98,16 +99,25 @@ export const submitApplication =
...
     // FIXME: this is temporarily until the round manager adds the encrypted field
     roundApplicationMetadata.applicationSchema.forEach((question) => {
-      if (/email/i.test(question.question.toLowerCase())) {
+      if (/email|cool/i.test(question.question.toLowerCase())) {
         // eslint-disable-next-line
         question.encrypted = true;
       }
     });
 
+    const { chainID } = state.web3;
```
Comment (codenamejason) [inline client/src/actions/roundApplication.ts:108]: we can update this with wagmi after we merge

File: client/src/services/lit.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import { Buffer } from "buffer";
+import LitJsSdk from "lit-js-sdk";
+
+// @ts-ignore
+window.Buffer = Buffer;
+
+const client = new LitJsSdk.LitNodeClient();
+
+const ROUND_OPERATOR =
```
Comment (codenamejason) [inline client/src/services/lit.ts:9]: assuming this is the 'hardcoded' value you were referring to to be able to test

**Review by codenamejason — APPROVED** [review]

**Review by codenamejason — APPROVED** [review]
All looks good

File: client/src/services/lit.ts (hunk 2 above)
Comment (gravityblast) [reply client/src/services/lit.ts:9]: yeah this is the hash of the role so it should be always that one

**Pull request merged**